app/editor/src/store/hooks/lookup/useLookup.ts

```diff
@@ -434,6 +434,7 @@ export const useLookup = (): [ILookupState, ILookupController] => {
       init: async () => {
         // TODO: Handle failures
         await controller.getLookups();
+        store.storeIsReady(true);
```

Comment:
We now have a new property to know when the lookups have successfully been received.  Use it to block certain useEffects from occuring.